packages/flutter/lib/src/rendering/sliver_fixed_extent_list.dart

```diff
@@ -40,44 +40,51 @@ abstract class RenderSliverFixedExtentBoxAdaptor extends RenderSliverMultiBoxAda
   ///
   /// The [childManager] argument must not be null.
   RenderSliverFixedExtentBoxAdaptor({
+    double itemSpacing: 0.0,
     @required RenderSliverBoxChildManager childManager,
-  }) : super(childManager: childManager);
+  }) : _itemSpacing = itemSpacing, super(childManager: childManager) {
+    assert(itemSpacing != null);
+  }
 
   /// The main-axis extent of each item.
   double get itemExtent;
 
-  /// The layout offset for the child with the given index.
+  /// How much empty space to place between each child in the main axis.
```

Comment:
I think it would be worth making it clear that this space only exists in between items, not at the beginning or the end of the list ... because it's not the same as just wrapping each item in 

```
Padding(padding: new EdgeOffsets.only(bottom: itemSpacing),  child: item)
```